src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4401,9 +4401,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_FeatureNotAvailableInVersion7" xml:space="preserve">
     <value>Feature '{0}' is not available in C# 7.0. Please use language version {1} or greater.</value>
   </data>
-  <data name="ERR_FeatureIsExperimental" xml:space="preserve">
```

Comment:
Please do not remove this from code from the compiler. It is intended to facilitate experimental code residing in branches of the compiler, even though the product code does not contain experiments.